### ISSUE TITLE
Add pagination in results main page

### ISSUE THREAD

**Issue by tushar-rishav** [coala-analyzer/coala-html#74]

**Comment by sahilarora535** [comment]
How should the pagination be? Like, should we be able to select the number of items per page?

**Comment by hemangsk** [comment]
Since this is an angular web app, we can add  filters too, to filter results based on Bears, Severity etc

**Comment by sahilarora535** [comment]
Yes, can be done. But first let's concentrate on pagination itself. How do you suggest pagination be implemented? Do we need to add a field for user to select number of options.

**Comment by KVGarg** [comment]
@tushar-rishav I would like to add pagination feature. Can you assign this issue to me !!
Afterwards, we can add filters on results too.
